lib/tasks/case_contact_types_reminder.rb

```diff
@@ -0,0 +1,83 @@
+class CaseContactTypesReminder
+  FIRST_MESSAGE = "It's been 60 days or more since you've reached out to these members of your youth's network:"
+  THIRD_MESSAGE = "If you have made contact with them in the past 60 days, remember to log it: [link to create new case contact for assigned case]"
```

Comment:
What do you guys think is the best way to put a link to `/case_contacts/new` in this string?

Reply:
You'd need to send the full url in the SMS. So something like "https://casa-qa.herokuapp.com/case_contacts/new". However, make sure to shorten the url's using the short.io class before sending.

Reply:
Shouldn't we use a dynamic root url instead? So it works for all dev, test, qa, and production?

Reply:
yeah, dynamic url does sound better actually. I'm not sure how to implement it in rails.

Reply:
Might postpone this to a future ticket.